mne/io/pick.py

```diff
@@ -73,6 +73,12 @@ def get_channel_type_constants(include_defaults=False):
                     kind=FIFF.FIFFV_FNIRS_CH,
                     unit=FIFF.FIFF_UNIT_V,
                     coil_type=FIFF.FIFFV_COIL_FNIRS_CW_AMPLITUDE),
+                fnirs_fd_ac_amplitude=dict(
+                    kind=FIFF.FIFFV_FNIRS_CH,
```

Comment:
Should this have a unit?

Reply:
add a unit here only if it will *always* be the same for this kind of channel.  (e.g., "ref_meg" channels can be different coil_types that have different units depending on the MEG machine the data was recorded on, so no unit is defined for the "ref_meg" entry here).  If the unit can vary, but there is a sensible default unit that you want to use in most cases (e.g., when generating synthetic data), add it below in the `defaults` dictionary (under `if include_defaults:`).

Reply:
I think this will always be V and the other radians, I'll push a commit for this